src/board/board.ts

```diff
@@ -1,26 +1,31 @@
 import chalk from "chalk";
 import { renderTile, Tile } from "../tile/tile";
+import { checkIfMoveIsAllowed } from "../move/move";
 
 export type NullableTile = Tile | undefined;
 export type Board = NullableTile[][];
 
 const BLANK_CHAR = chalk.black(" ");
+let isFirstRender = false;
 
-export const renderLine = (lines: Tile[]): string => {
+export const renderLine = (
+  lines: NullableTile[],
+  lineIndex: number
+): string => {
   const line = "";
+  let x = 0;
```

Comment:
I don't believe `x` is used

Reply:
I removed it.

Reply:
I still see it

Reply:
Same, it should be deleted, along with all usage of x in the function.